hack/test-prerequisites.go

```diff
@@ -27,8 +27,8 @@ func main() {
 
 	client := apiext.NewForConfigOrDie(cfg)
 	for _, path := range []string{
-		"vendor/github.com/openshift/api/config/v1/0000_00_cluster-version-operator_01_clusterversion-Default.crd.yaml",
-		"vendor/github.com/openshift/api/config/v1/0000_00_cluster-version-operator_01_clusteroperator.crd.yaml",
+		"vendor/github.com/openshift/api/config/v1/zz_generated.crd-manifests/0000_00_cluster-version-operator_01_clusterversion-Default.crd.yaml",
```

Comment:
```suggestion
		"vendor/github.com/openshift/api/config/v1/zz_generated.crd-manifests/0000_00_cluster-version-operator_01_clusterversions-Default.crd.yaml",
```

Reply:
Thanks!  Fixed in dbe139c03af347b -> e9762c607d5456b1 with:

```console
$ sed -i 's/\(clusterversion\|clusteroperator\)/\1s/' hack/test-prerequisites.go
```

so it will be easy to apply to the 4.16.z backport.